CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This file is used to list changes made in each version of the lvm cookbook.
 
 ## Unreleased
 
+## 6.1.1 - *2022-09-29*
```

Comment:
Can you remove this? The pipeline automatically takes anything under `## Unreleased` and puts it under the version number that the pipeline determines the cookbook should be incremented to.

Reply:
Will do